### ISSUE TITLE
[BUG] Market block steals emeralds.

### ISSUE THREAD

**Issue by sspazzy** [CalaMariGold/Rebirth-Of-The-Night#94]
ISSUES NOT USING OUR TEMPLATE WILL NOT BE CONSIDERED AND WILL BE CLOSED!

**Describe the bug**
If you have emeralds in the market payment slot and click an item in the JEI interface, the emeralds will disappear completely. 

**To Reproduce**
Steps to reproduce the behavior: Put emerald in payment slot of market block. Click on JEI item. go back to market interface. Emeralds disappeared.

**Expected behavior**
Select the intended item to buy from the JEI interface with the emeralds in the payment slot ready to go. Buy intended amount of item from market using the intended amount of emeralds.

**Screenshots**

**Info (please complete the following information):**
 - Modpack Version: 2.76.1
 - Server or Singleplayer: Singleplayer
 - Version the world was created on: 2.76.1

**Additional context**
Very fast tossing item on the ground blip when you click on the JEI interface as if the emeralds are getting sucked into the market from the player. Too fast to clearly see and does't show in 3rd person.

**Comment by CalaMariGold** [comment]
We can disable the market block until this is fixed.

**Comment by sspazzy** [comment]
> We can disable the market block until this is fixed.

**Comment by deebyte** [comment]
The market block has been a little buggy for a long time in Pam's 1.12, and I'm not sure how quickly this specific bug might be fixed. As the market block is somewhat essential to make large-scale farming worthwhile, it would be a major loss to have it disabled indefinitely due to a bug like the above (which, while reproducible, is easily avoided). Just food for thought.

**Comment by CalaMariGold** [comment]
Good point. We will include a JEI description about the bug then and how to avoid it.

**Comment by kil0byt3** [comment]
Done, desc added
